core/src/com/unciv/ui/screens/victoryscreen/VictoryScreenIllustrations.kt

```diff
@@ -243,6 +245,14 @@ class VictoryScreenIllustrations(
                     total += milestone.params[0].toInt()
                     civ.policies.completedBranches.size
                 }
+                MilestoneType.MoreCountableThanEachPlayer -> {
+                    total += if (selectedCiv.shouldHideCivCount()) game.gameParameters.maxNumberOfPlayers
+                        else game.civilizations.count { it != civ && it.isMajorCiv() && it.isAlive() }
+                    game.civilizations.count {
+                        it != civ && it.isMajorCiv() && it.isAlive() &&
+                        (Countables.getCountableAmount(milestone.params[0], GameContext(civ)) ?: 0) > (Countables.getCountableAmount(milestone.params[1], GameContext(it)) ?: 0)
```

Comment:
This is the third time we have the same logic.
What I'm getting from this is that we need *in MilestoneType.MoreCountableThanEachPlayer* a function / some functions to get this common data
That we can then call with e.g. MilestoneType.MoreCountableThanEachPlayer.getStatus(civ: Civilization)

Reply:
That's a great idea.